jit/v2/JIT_v2.0.0.yaml

```
@@ -3377,15 +3377,15 @@ components:
           description: |
             The maximum length of a ship's hull measured parallel to the waterline (Length OverAll).
 
-            If the length is specified in feet (`FOT`) then the decimal part should be considered as a fraction of a foot and **not** as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6")
+            If the length is specified in feet (`FOT`) then the decimal part should be considered as a fraction of a foot and **not** as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6").
```

```suggestion
            If the length is specified in feet (`FOT`) then the decimal part means a fraction of a foot and **not** as a number of inches. E.g. 120.5 feet means 120 and a half foot (which would be 120'6").
```
To avoid the `should be considered` which can be read as optional interpretation. If you agree, or if you have a better idea, please apply it at all those lines, having this construct.

---

Sounds like a good idea - fixed